src/main/java/core/basesyntax/utility/FruitQuantityChecker.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.utility;
+
+import java.util.Map;
+
+public class FruitQuantityChecker {
+    public void checkFruitQuantity(Map<String, Integer> fruitMap) {
+        for (Integer temp : fruitMap.values()) {
+            if (temp < 0) {
```

Comment:
this will throw NPE if integer was null, if you plan to validate the storage after processing data, maybe this would be a good idea to check for too

Reply:
+